<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find asr_intermediate_object_generator)/log/rosconsole.config" />
    <node name="asr_intermediate_object_generator" pkg="asr_intermediate_object_generator" type="asr_intermediate_object_generator" output="screen">
       <rosparam command="load" file="$(find asr_intermediate_object_generator)/rsc/Domain.yaml" />
       <!-- Needed to get the dbfilename -->
       <rosparam file="$(find asr_recognizer_prediction_ism)/launch/helpers/scene_recognition.yaml"/>
       <!-- XXX will be replaced with /rp_ism_node/dbfilename -->
       <param name="AutomatPath" value="$(find asr_state_machine)/rsc/IntermediateObject_XXX.xml"/>
       <param name="LogPath" value="$(find asr_intermediate_object_generator)/logs/"/>
       <!-- XXX will be replaced with /rp_ism_node/dbfilename -->
       <param name="NBVPath" value="$(find asr_world_model)/rsc/IntermediateObject_XXX.xml"/>
    </node>   
</launch>
